packages/gestalt/src/TableHeader.js

```diff
@@ -4,14 +4,21 @@ import cx from 'classnames';
 import styles from './Table.css';
 
 type Props = {|
+  /**
+   * Must be an instance of Table.Row. See the [Subcomponent section](https://gestalt.pinterest.systems/table#Subcomponents) to learn more.
+   */
   children: Node,
+  /**
+   * If true, the table header will be sticky and the table body will be scrollable. See the [sticky Header](https://gestalt.pinterest.systems/table#Sticky-header) and the [sticky header and columns](https://gestalt.pinterest.systems/table#v) variants for details.
```

Comment:
looks like something happened to the "sticky header and columns" link

Reply:
DONE